### ISSUE TITLE
Application stops with SyntaxError: Unexpected token L

### ISSUE THREAD

**Issue by mcwieger** [empierre/MyDomoAtHome#158]
After running stable for a day or three, MDAH stopped with the following error:

```
SyntaxError: Unexpected token L
    at Object.parse (native)
    at Request._callback (/volume1/@appstore/MyDomoAtHome/app/mdah.js:2358:29)
    at Request.self.callback (/volume1/@appstore/MyDomoAtHome/app/node_modules/request/request.js:200:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (/volume1/@appstore/MyDomoAtHome/app/node_modules/request/request.js:1067:10)
    at emitOne (events.js:82:20)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (/volume1/@appstore/MyDomoAtHome/app/node_modules/request/request.js:988:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
```

See full error log attached.

[mydomoathome.log](https://github.com/empierre/MyDomoAtHome/files/1534378/mydomoathome.log)

**Comment by empierre** [comment]
Hello,

  This is domoticz that sent an incomplete messagen, strange it crashes ? does synology auto restart it ?

**Comment by mcwieger** [comment]
No, it never restarts. Would be nice to have a script that does that, but have not found a how-to yet.

**Comment by empierre** [comment]
will have a look how domoticz do so, on PI it does through a process called
forever.

**Comment by mcwieger** [comment]
It stopped again, same error but with token P. Rest of the message is the same. Looking at the rest of the file, it's my P1 smart meter causing it, right? I suppose I can live without that info in Imperihome, but I can imagine that's not satisfactory for you, knowing this will cause it to crash?

**Comment by empierre** [comment]
in the initial log file, the last update is at the beginning of the file, if you have more clue of something I could reproduce I'll look into it.

**Comment by empierre** [comment]
try from line 2359:
var data 
try {
  data= JSON.parse(body);
} catch (e) {
    return console.error(e);
  }

**Comment by mcwieger** [comment]
Resolved in version 0.2.36.
